package.json

```diff
@@ -111,6 +111,7 @@
 		"google-protobuf": "^3.21.0",
 		"json-bigint": "^1.0.0",
 		"reflect-metadata": "^0.1.13",
-		"typescript": "^4.7.2"
+		"typescript": "^4.7.2",
+		"app-root-path": "^3.1.0"
```

Comment:
should library take this responsibility or delegate it to the app?

Reply:
Next/Vercel frameworks do this automatically for you and follow the same sequence. In an ideal world we should let user do it, but doing it ourselves should not have any side-effects for 90% of the use cases (like where they have a different load order for files). 

Doing this would simplify our [setup scripts](https://github.com/tigrisdata/tigris-vercel-starter/blob/main/scripts/setup.ts). 

@ovaistariq your thoughts?